src/en/news/blog/2025/crimson-balance-cpu-part1/index.md

```diff
@@ -0,0 +1,832 @@
+
+```bash
+crimson_be_table["cyan"]="--cyanstore"
+crimson_be_table["sea"]="--seastore --seastore-devs ${STORE_DEVS}"
+crimson_be_table["blue"]="--bluestore --bluestore-devs ${STORE_DEVS}"
+```
+Once the execution completes, the results are archived in .zip files according
+to workload, saved in the output directory (`/tmp` by default, which can be
+provided with option `-d`).
+
+<details>
+<summary>Click to see a snapshot of the generated results.</summary>
+
+```bash
+cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_randread.zip      cyan_8osd_5reactor_8fio_bal_socket_rc_1procs_seqread.zip   cyan_8osd_6reactor_8fio_bal_osd_rc_1procs_randread.zip      cyan_8osd_6reactor_8fio_bal_socket_rc_1procs_seqread.zip
+cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_randwrite.zip     cyan_8osd_5reactor_8fio_bal_socket_rc_1procs_seqwrite.zip  cyan_8osd_6reactor_8fio_bal_osd_rc_1procs_randwrite.zip     cyan_8osd_6reactor_8fio_bal_socket_rc_1procs_seqwrite.zip
+cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_seqread.zip
+```
+</details>
+
+<!-- look inside ranread for example, which are the important output .json, etc.
+-->
+Each archive contains the result output files and measurements from that
+workload execution.
+
+<details>
+<summary>Click to see inside a results .zip file.</summary>
+The files more important are:
+
+- `cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_randread.json`: the (combined) FIO
+output file, which contains the I/O throughput and latency measurements. It
+contains integrated the CPU and MEM utilisation from the OSD process as well.
+    - `cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_randread_cpu_avg.json`: the
+    OSD and FIO CPU and MEM utilisation averages. These have been collected
+    from the OSD process and the FIO client process via top (30 samples over 5
+    minutes).
+    - `cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_randread_diskstat.json`: the
+    diskstat output. A sample is taken before and after the test, the .json
+    contains the differences.
+    - `cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_randread_top.json`: the output
+    from top, parsed via jc to produce a .json. **Note**: jc does not yet
+    support individual CPU core utilisation, so we have to rely on the overall
+    CPU utilisation (per thread).
+    - `new_cluster_dump.json`: the output from `ceph tell ${osd_i}
+    dump_metrics` command, which contains the individual OSD performance
+    metrics.
+    - `FIO_cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_randread_top_cpu.plot`:
+    the plot mechanically generated from the top output, showing the CPU
+    utilisation over time for the FIO client.
+    - `OSD_cyan_8osd_5reactor_8fio_bal_osd_rc_1procs_randread_top_mem.plot`:
+    the plot mechanically generated from the top output, showing the MEM
+    utilisation over time for the OSD process.
+
+</details>
+
+To produce the post-processing and side-by-side comparison, the following script
+is run: the arguments are the input directory that contains the runs we want to
```

Comment:
I think we can move the description of the arguments below the command to run the script.